composer/utils/inference.py

```diff
@@ -29,10 +30,21 @@
 
 log = logging.getLogger(__name__)
 
-__all__ = ['export_for_inference', 'ExportFormat', 'export_with_logger']
+__all__ = ['export_for_inference', 'ExportFormat', 'export_with_logger', 'quantize_dynamic']
 
 Transform = Callable[[nn.Module], nn.Module]
 
+# This is the most common way to use dynamic quantization.
+#  Example:
+#    from composer.utils import quantize_dynamic
+#    export_for_inference(
+#        ...
+#        transforms = [quantize_dynamic],
+#        ...
+#    )
+#  A user can always redefine it with extra options. This also serves as an example of what to pass to transforms.
+quantize_dynamic = functools.partial(torch.quantization.quantize_dynamic, qconfig_spec={torch.nn.Linear})
```

Comment:
Is there a reason we only apply this to linear layers?

Reply:
Dynamic quantization only works for linear layers and LSTMs. For us, only linears are important.

Reply:
Got it; thanks.